docs/pages/workflow/expo-cli.md

```diff
@@ -1016,13 +971,104 @@ Alias: `expo ui`
+| `expo prebuild`      | `npx expo prebuild`    | Versioned                                     |
+| `expo customize:web` | `npx expo customize`   | Versioned, and renamed                        |
+| `expo build:web`     | `npx expo export:web`  | Versioned, and renamed                        |
+| `expo eject`         | `npx expo prebuild`    | Merged into `npx expo prebuild`               |
+| `expo start:web`     | `npx expo start`       | Merged into `npx expo start`                  |
+
+### Deprecated
+
+| Legacy                        | Notes                           |
+| ----------------------------- | ------------------------------- |
+| `expo client:ios`             | Removed in favor of Dev Clients |
+| `expo send`                   | Unimplemented                   |
+| `expo client:install:ios`     | Unimplemented                   |
+| `expo client:install:android` | Unimplemented                   |
+| `expo doctor`                 | Undecided                       |
+| `expo upgrade`                | Undecided                       |
+
+### Services
+
+| Legacy                           | New                     | Notes              |
+| -------------------------------- | ----------------------- | ------------------ |
+| `expo publish`                   | `eas update`            | Moved to `eas-cli` |
+| `expo publish:set`               | `eas update`            | Moved to `eas-cli` |
+| `expo publish:rollback`          | `eas update`            | Moved to `eas-cli` |
+| `expo publish:history`           | `eas update`            | Moved to `eas-cli` |
+| `expo publish:details`           | `eas update`            | Moved to `eas-cli` |
+| `expo credentials:manager`       | `eas credentials`       | Moved to `eas-cli` |
+| `expo fetch:ios:certs`           | `eas XXX`               | Moved to `eas-cli` |
+| `expo fetch:android:keystore`    | `eas XXX`               | Moved to `eas-cli` |
+| `expo fetch:android:hashes`      | `eas XXX`               | Moved to `eas-cli` |
+| `expo fetch:android:upload-cert` | `eas XXX`               | Moved to `eas-cli` |
+| `expo push:android:upload`       | `eas XXX`               | Moved to `eas-cli` |
+| `expo push:android:show`         | `eas XXX`               | Moved to `eas-cli` |
+| `expo push:android:clear`        | `eas XXX`               | Moved to `eas-cli` |
+| `expo url`                       | `eas XXX`               | Moved to `eas-cli` |
+| `expo url:ipa`                   | `eas XXX`               | Moved to `eas-cli` |
+| `expo url:apk`                   | `eas XXX`               | Moved to `eas-cli` |
+| `expo webhooks`                  | `eas XXX`               | Moved to `eas-cli` |
+| `expo webhooks:add`              | `eas XXX`               | Moved to `eas-cli` |
+| `expo webhooks:remove`           | `eas XXX`               | Moved to `eas-cli` |
+| `expo webhooks:update`           | `eas XXX`               | Moved to `eas-cli` |
```

Comment:
I don't know what the new commands are for this

Reply:
- all `expo fetch` and `push` commands were moved to `eas credentials` - but no longer specific commands for each action, it's an interactive prompt.
- `expo url` / `expo url:ipa` / `expo url:apk` don't exist anymore. `eas build:list` and `eas build:view` are replacements. can use the `--json` flag with jq and `build:list` filters to pick the build whose data you want to access:

```
╰─$ eas build:list --help
list all builds for your project

USAGE
  $ eas build:list [--platform all|android|ios] [--json] [--status new|in-queue|in-progress|errored|finished|canceled] [--distribution
    store|internal|simulator] [--channel <value>] [--appVersion <value>] [--appBuildVersion <value>] [--sdkVersion <value>] [--runtimeVersion
    <value>] [--appIdentifier <value>] [--buildProfile <value>] [--gitCommitHash <value>] [--limit <value>]

FLAGS
  --appBuildVersion=<value>
  --appIdentifier=<value>
  --appVersion=<value>
  --buildProfile=<value>
  --channel=<value>
  --distribution=(store|internal|simulator)
  --gitCommitHash=<value>
  --json                                                         Enable JSON output, non-JSON messages will be printed to stderr
  --limit=<value>
  --platform=(all|android|ios)
  --runtimeVersion=<value>
  --sdkVersion=<value>
  --status=(new|in-queue|in-progress|errored|finished|canceled)
```
 
- webhooks:

```
╰─$ eas webhook --help
manage webhooks

USAGE
  $ eas webhook:COMMAND

COMMANDS
  webhook:create  create a webhook
  webhook:delete  delete a webhook
  webhook:list    list webhooks
  webhook:update  update a webhook
  webhook:view    view a webhook
```